pkg/reconciler/layer3.go

```diff
@@ -63,21 +63,22 @@ func (r *reconcile) reconcileLayer3(l3vnis []networkv1alpha1.VRFRouteConfigurati
 		return allConfigs[i].VNI < allConfigs[j].VNI
 	})
 
-	created, err := r.reconcileL3Netlink(l3Configs)
+	created, deletedVRF, err := r.reconcileL3Netlink(l3Configs)
 	if err != nil {
 		r.Logger.Error(err, "error reconciling Netlink")
 		return err
 	}
 
-	err = r.reconcileTaasNetlink(taasConfigs)
+	deletedTaas, err := r.reconcileTaasNetlink(taasConfigs)
 	if err != nil {
 		return err
 	}
+	reloadTwice := deletedVRF || deletedTaas
```

Comment:
why do we need to reload twice when a vrf is removed?

Reply:
First reload removes all statements but does not remove `router bgp <asn> vrf <vrf>` statement. This is achieved by a second reload.

Reply:
👍